sdk/formrecognizer/azure-ai-formrecognizer/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,639 @@
+# Guide for migrating azure-ai-formrecognizer to version 3.2.x from versions 3.1.x and below
+
+This guide is intended to assist in the migration to `azure-ai-formrecognizer (3.2.x)` from versions `3.1.x` and below. It will focus on side-by-side comparisons for similar operations between versions. Please note that version `3.2.0b1` will be used for comparison with `3.1.2`.
+
+Familiarity with `azure-ai-formrecognizer (3.1.x and below)` package is assumed. For those new to the Azure Form Recognizer client library for Python please refer to the [README][readme] rather than this guide.
+
+## Table of Contents
+- [Migration benefits](#migration-benefits)
+- [Important changes](#important-changes)
+    - [Client usage](#client-usage)
+    - [Analyzing document](#analyzing-documents)
+    - [Analyzing a custom model](#analyzing-a-custom-model)
+    - [Training a custom model](#training-a-custom-model)
```

Comment:
Building a custom model?

Reply:
I left it as training since the guide is catering more towards people coming from the previous version. I commented on this here: https://github.com/Azure/azure-sdk-for-python/pull/20960#discussion_r719703484